### PR TITLE
Remove unnecessary code in `created` method.

### PR DESCRIPTION
**Pull request by 2ykwang**
related #404 

already run the same code in the Django
ref: https://github.com/django/django/blob/3.2/django/contrib/admin/utils.py#L381-L429

```python
# ...
elif isinstance(field, models.DateTimeField):
    return formats.localize(timezone.template_localtime(value))
# ...
```

### CODE REVIEW

**Comment by alieh-rymasheuski** [comment]
@2ykwang, the main part of the `created` method is a call to `localtime`. It matters when the server time is not UTC (it's set to NY time in my case).

These are the outputs of your version, the current version, and a compromise version which delegates the formatting to Django code:
```python
    def created(self, obj):
        return obj.timestamp
```
Created: `2022-10-03 16:09:59.723996+00:00`
```python
    def created(self, obj):
        return localtime(obj.timestamp).strftime("%Y-%m-%d %H:%M:%S")
```
Created: `2022-10-03 12:09:59`
```python
    def created(self, obj):
        return localtime(obj.timestamp)
```
Created: `2022-10-03 12:09:59.723996-04:00`

I would argue that precision to seconds and implicit server-local timezone are beneficial to the readability.

**Comment by 2ykwang** [comment]
@alieh-rymasheuski When the `created` method returns a `DateTimeField` object, it displays the data formatted according to Django settings. in my opinion, it felt awkward because the `LogEntryMixin.created` date format was different from other Django Admin date formats.

**Comment by alieh-rymasheuski** [comment]
Agree. But please make sure to keep the `localtime` call in place - it is valuable for the case when the server timezone is different from UTC.

**Comment by 2ykwang** [comment]
@alieh-rymasheuski 
thank you I just edited the code.

**Review by alieh-rymasheuski — APPROVED** [review]

**Comment by alieh-rymasheuski** [comment]
@2ykwang, please add a changelog entry as well.

@hramezani, would you like to take a look too?

**Comment by 2ykwang** [comment]
@alieh-rymasheuski done!

File: auditlog/mixins.py [hunk 1]
```diff
@@ -16,11 +17,11 @@
 
 
 class LogEntryAdminMixin:
+    @admin.display(description="created")
```
Comment (hramezani) [inline auditlog/mixins.py:20]:
```suggestion
    @admin.display(description="Created")
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 #### Fixes
 
+- fix: Remove unnecessary code in `created` method. ([#438](https://github.com/jazzband/django-auditlog/pull/438))
```
Comment (hramezani) [inline CHANGELOG.md:10]: I think we don't need to mention this here as it's not adding/changing a feature.
Please remove it.

**Comment by hramezani** [comment]
Thanks @2ykwang for the patch and @alieh-rymasheuski for review!

**Pull request merged**